mmhuman3d/data/data_structures/smc_reader.py

```diff
@@ -567,26 +590,67 @@ def _get_keypoints2d(self, device, device_id, frame_id):
 
     def get_kinect_keypoints2d(self, device_id, frame_id=None):
         assert self.num_kinects > device_id >= 0
-        return self._get_keypoints2d('Kinect', device_id, frame_id)
+        return self.get_keypoints2d('Kinect', device_id, frame_id)
 
     def get_iphone_keypoints2d(self, device_id, frame_id=None):
         assert device_id >= 0
-        return self._get_keypoints2d('iPhone', device_id, frame_id)
+        return self.get_keypoints2d('iPhone', device_id, frame_id)
 
-    def get_keypoints_num(self):
-        return self.keypoints_num
+    def get_color(self, device, device_id, frame_id=None, disable_tqdm=False):
```

Comment:
We may add a unit test for this func

Reply:
Unit test added in https://github.com/open-mmlab/mmhuman3d/pull/73/commits/f943649f4a1fd45aa0e9e7134989f0b4c127b31e.